packages/config-loader/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@backstage/config-loader",
   "description": "Config loading functionality used by Backstage backend, and CLI",
-  "version": "0.8.0",
+  "version": "0.9.0",
```

Comment:
Good news is versions are not manually updated; they are taken care of by the changeset process, so you can leave this out.  More info here: https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets

Reply:
Sure. Thanks